jenkins_jobs/periodic_pipelines.groovy

```diff
@@ -1,27 +0,0 @@
-def createPeriodicPipeline(String name, String cronExpression) {
-  pipelineJob(name) {
-    definition {
-      cps {
-        script("""\
-node('master') {
-  dir('infrastructure') {
-    git(credentialsId: 'github-user-pass-credentials',
-        url: 'https://$GITHUB_DOMAIN/$GITHUB_ORGANIZATION_NAME/infrastructure.git',
-        branch: '$REPOSITORY_COMMIT')
-  }
-  pipeline = load 'infrastructure/pipeline/${name}.groovy'
-}
-
-pipeline.execute()
-""")
-        sandbox()
-      }
-    }
-    triggers {
-      cron(cronExpression)
-    }
-  }
-}
-
-createPeriodicPipeline('devel', "$NIGHTLY_BUILDS_CRON_EXPRESSION")
-createPeriodicPipeline('release', "$NIGHTLY_BUILDS_CRON_EXPRESSION")
```

Comment:
I was thinking more of removing the `createPeriodicPipeline` line for devel and just disabling the release pipeline in the Jenkins interface. We will have to run the release pipeline from time to time won't we?

Reply:
That works too.